tests/model/model.cpp

```diff
@@ -628,6 +628,106 @@ TEST(Model, removeComponentInsensitiveToOrder)
     EXPECT_EQ(size_t(2), modelApi->componentCount());
 }
 
+TEST(Model, cleanCreatedModel)
+{
+    auto model = libcellml::Model::create();
+    auto c1 = libcellml::Component::create("c1");
+    auto c2 = libcellml::Component::create();
+    c2->setId("c2");
+    auto c3 = libcellml::Component::create();
+    auto c4 = libcellml::Component::create();
+    auto c5 = libcellml::Component::create();
+    auto c6 = libcellml::Component::create();
+    auto u1 = libcellml::Units::create("used");
+    auto u2 = libcellml::Units::create();
+    u2->setId("u2");
+    auto u3 = libcellml::Units::create();
+    auto v = libcellml::Variable::create("x");
+
+    model->addComponent(c1);
+    model->addComponent(c2);
+    model->addComponent(c3);
+    model->addComponent(c4);
+
+    // test unnamed components in an encapsulation hierarchy
```

Comment:
`// Test unnamed components in an encapsulation hierarchy.`

Reply:
Tests changed, so that comment is no longer relevant.